### PR TITLE
Fix running of examples on Travis

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ env:
 
 script:
   - echo "build --show_progress_rate_limit=5.0" >> .bazelrc
+  - set -o errexit
```
Comment (anghelcovici) [inline .travis.yml]: Heh..

File: scripts/run_examples [hunk 2]
```diff
@@ -13,7 +13,7 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 "$SCRIPTS_DIR/build_server_docker"
 readonly SERVER_CONTAINER_ID=$("$SCRIPTS_DIR/docker_run" \
   --detach \
-  ./bazel-bin/oak/server/oak)
+  ./bazel-bin/oak/server/asylo/oak)
```
Comment (anghelcovici) [inline scripts/run_examples]: :+1:

**Comment by daviddrysdale** [comment]
Hmm, Travis is failing with an error I've not seen before:
```
2019-08-20 15:16:14  QFATAL  manager_client.h : 51 : Failed: 8/::asylo::error::SgxErrorSpace::Out of memory: Failed to create an enclave/
```
But on the plus side, the test failure did trigger a Travis build failure&hellip;

**Comment by daviddrysdale** [comment]
Added a commit to run each example separately, so memory limits don't get hit - PTAL.

**Comment by tiziano88** [comment]
I would prefer we solve the node lifetime issue, even if it's just an RPC to shut down a node once we are done with it. I just created #181 to track that.

**Comment by daviddrysdale** [comment]
Yep, I agree we need a strategy for node termination (but it would be good to get the tests running more reliably in the meanwhile).

**Pull request merged**